openquake/hazardlib/calc/hazard_curve.py

```diff
@@ -24,26 +24,15 @@
 .. code-block:: python
 
    import sys
-   import logging
-   from openquake.baselib import parallel
-   from openquake.hazardlib.calc.filters import SourceFilter
-   from openquake.hazardlib.calc.hazard_curve import calc_hazard_curves
-   from openquake.commonlib import readinput
+   from openquake.commonlib import logs
+   from openquake.calculators.base import get_calc
 
    def main(job_ini):
-       logging.basicConfig(level=logging.INFO)
-       oq = readinput.get_oqparam(job_ini)
-       sitecol = readinput.get_site_collection(oq)
-       src_filter = SourceFilter(sitecol, oq.maximum_distance)
-       csm = readinput.get_composite_source_model(oq, srcfilter=src_filter)
-       for sm in csm.full_lt.sm_rlzs:
-           for rlz in csm.full_lt.get_rlzs(sm.ordinal):
-               gsim_by_trt = csm.full_lt.gsim_by_trt(rlz)
-               hcurves = calc_hazard_curves(
-                   sm.src_groups, src_filter, oq.imtls,
-                   gsim_by_trt, oq.truncation_level,
-                   parallel.Starmap.apply)
-           print('rlz=%s, hcurves=%s' % (rlz, hcurves))
+       calc_id = logs.init()
+       calc = get_calc(job_ini, calc_id)
+       calc.run(individual_curves='true', shutdown=True)
+       print('The hazard curves are in %s::/hcurves-rlzs'
+             % calc.datastore.filename)
```

Comment:
This is a shortcut :) it would be nice having a script that allows the user to access the various realisations admitted by the logic tree.

Reply:
Hopefully I will get there